Products/CMFPlone/browser/search.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from DateTime import DateTime
 from plone.app.contentlisting.interfaces import IContentListing
+from plone.base.interfaces.siteroot import IPloneSiteRoot
```

Comment:
```suggestion
from Products.CMFPlone.interfaces import IPloneSiteRoot
```

in Plone 5.2 there is no `plone.base`